src/components/Team/UpdateProfile.js

```diff
@@ -153,23 +135,53 @@ const UpdateProfile = props => {
 			if (!err) {
 				try {
 					const formData = new FormData();
-					if (imageLink !== null) {
-						formData.append("image", imageLink);
+					if (values.image.file) {
+						formData.append(
+							"image",
+							values.image.file.originFileObj
+						);
+					} else {
+						formData.append("image", values.image);
+					}
+					formData.append("name", values.name);
+					formData.append("email", values.email);
+					if (
+						values.password !== undefined &&
+						values.password !== ""
+					) {
+						formData.append("password", values.password);
+					}
+					if (values.contact !== undefined && values.contact !== "") {
+						formData.append("contact", values.contact);
+					}
+					formData.append("designation", values.designation);
+					// formData.append("dob", dob);
+					if (values.github !== undefined && values.github !== "") {
+						formData.append("github", values.github);
+					}
+					if (
+						values.linkedin !== undefined &&
+						values.linkedin !== ""
+					) {
+						formData.append("linkedin", values.linkedin);
+					}
+					if (values.twitter !== undefined && values.twitter !== "") {
+						formData.append("twitter", values.twitter);
+					}
+					if (
+						values.portfolio !== undefined &&
+						values.portfolio !== ""
+					) {
+						formData.append("portfolio", values.portfolio);
 					}
-					formData.append("name", name);
-					formData.append("email", email);
-					formData.append("password", password);
-					formData.append("contact", contact);
-					formData.append("designation", designation);
-					formData.append("dob", dob);
-					formData.append("github", github);
-					formData.append("linkedin", linkedin);
-					formData.append("twitter", twitter);
-					formData.append("portfolio", portfolio);
 
 					const res = await updateUserService(formData);
 					if (res.message === "success") {
 						_notification("success", "Success", "Event Updated");
+						if (newName !== null || newEmail !== null) {
```

Comment:
This is not required on the frontend. User token expires when they change their name and email. They will get a session expired message when they try to access authenticated material. You need to catch that in the interceptor.